optaweb-vehicle-routing-benchmark/src/main/java/org/optaweb/vehiclerouting/benchmark/StopWatch.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.vehiclerouting.benchmark;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+class StopWatch {
+
+    private final long startTime;
+    private long stopTime;
+    private long lastLap;
+    private int lapCount = 0;
+    private final List<Lap> laps = new ArrayList<>(1010);
```

Comment:
Why is the ArrayList's capacity initialized with this number?

Reply:
Because the recommended number of locations to test is 1 - 1000. I originally wanted to measure the calculation time for a 1001th location being added to a matrix with 1000 locations so I made the list size slightly higher.

Reply:
Let's add a comment over there?